File: backend/src/main/java/cohappy/backend/model/dto/request/CreateChoreDTO.java
package cohappy.backend.model.dto.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

import java.time.LocalDate;
import java.util.List;
import java.util.Map;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CreateChoreDTO {
    private List<LocalDate> days;
    private Map<LocalDate, String> assignedTo;
    private String createdBy;
    private String houseCode;
    private String name;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/client/dto/request/CreateChoreDTO.kt
package cohappy.frontend.client.dto.request

import java.time.LocalDate

data class CreateChoreDTO(
    var days: List<LocalDate>? = null,
    var assignedTo: Map<LocalDate, String>? = null,
    var createdBy: String? = null,
    var houseCode: String? = null,
    var name: String? = null,
    var description: String? = null
)
